plugins/eventMacro/eventMacro/Runner.pm

```diff
@@ -1080,6 +1080,11 @@ sub next {
 						$eventMacro->set_full_hash($var->{real_name}, \%hash);
 					}
 					
+				} elsif ($var->{type} eq 'array' && $value =~ /^$macro_keywords_character(?:split)\(\s*(.*?)\s*\)$/) {
+					my ( $pattern, $scalar ) = map {/^\s*(.*?)\s*$/} parseArgs( "$1", undef, ',' );
+					my $scalar_var = find_variable( $scalar );
+					$self->error( "Scalar variable not recognized" ), return if !$scalar_var;
+					$eventMacro->set_full_array( $var->{real_name}, [ split $pattern, $eventMacro->get_scalar_var( $scalar_var->{real_name} ) ] );
```

Comment:
Maybe add support for any type of var? Like $array[0] and $hash{key} besides just $scalar.